src/hooks/use-swr-collection.ts

```diff
@@ -155,12 +91,11 @@ const createFirestoreRef = <Doc extends object = {}>(
     }
 
     if (where) {
-      function multipleConditions(w: WhereType<Doc>): w is WhereArray<Doc> {
-        return !!(w as WhereArray) && Array.isArray(w[0])
-      }
-      if (multipleConditions(where)) {
+      if (Serializer.multipleConditions<Doc>(where)) {
         where.forEach(w => {
-          ref = ref.where(w[0] as string | FieldPath, w[1], w[2])
```

Comment:
Why this change? Maybe I'm seeing the diffs wrong, but a date doesn't need to get added specially, correct? It just needs to be swapped in for `w[2]` by the serializer.

Reply:
Ah, that is a mistake on my part. It was the first thing I did, before realizing I wanted a better serializer. I'm reverting that change.